<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="onHomeButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>740</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrondissementButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>740</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PARIS </string>
    </property>
   </widget>
   <widget class="QPushButton" name="AboutUSButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>730</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>About Us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="JOButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>740</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SITES JO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>740</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMULATION</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLanguage">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>60</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>330</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Bienvenue à Paris, la ville lumière, connue pour son histoire riche, son architecture emblématique et son art incomparable. Nichée sur les rives de la Seine, Paris séduit avec ses célèbres monuments tels que la Tour Eiffel, le Louvre et la cathédrale Notre-Dame. Explorez les charmants arrondissements, dégustez une cuisine exquise et plongez-vous dans la culture artistique de cette métropole captivante.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Que vous soyez passionné de sport, d'histoire ou de culture, Paris a quelque chose à offrir à chacun. Profitez de l'atmosphère électrique des Jeux Olympiques et explorez la magie intemporelle de Paris, une ville qui continue de captiver et d'inspirer.&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playVideoButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenue :)</string>
    </property>
   </widget>
   <zorder>playVideoButton</zorder>
   <zorder>onHomeButton</zorder>
   <zorder>arrondissementButton</zorder>
   <zorder>AboutUSButton</zorder>
   <zorder>JOButton</zorder>
   <zorder>simButton</zorder>
   <zorder>comboBoxLanguage</zorder>
   <zorder>textEdit</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>graphicsView</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1356</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
